include/taily.hpp

```diff
@@ -137,10 +138,10 @@ struct Query_Statistics {
 [[nodiscard]] auto
 fit_distribution(Feature_Statistics const& query_term_stats) -> boost::math::gamma_distribution<>
 {
-    const double k = std::pow(query_term_stats.expected_value, 2.0)
-        / query_term_stats.variance;
-    const double theta =
-        query_term_stats.variance / query_term_stats.expected_value;
+    double epsilon = std::numeric_limits<double>::epsilon;
```

Comment:
`epsilon` is a member function of `numeric_limits`, you need to add `()`.

Reply:
Sorry, a typo while copy-pasting

Reply:
Btw, do you think it makes sense logically?

Reply:
It's certainly better than it was, so I think we can merge it. Let me think about this though, and see if we can have a more exact corner-case. But I need to look at the paper first to refresh my memory.